src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/Tag/Tag.jsx

```diff
@@ -37,9 +40,23 @@ const Tag = ({field, id, t}) => {
                             /* Do Nothing on blur BACKLOG-10095 */
                         }}
                         onChange={selection => {
+                            let adaptedSelection = [];
```

Comment:
Please use named variable instead of single char.
Move this code in an utility module and provide a test for it